server/src/services/appVersion.js

```diff
@@ -46,6 +48,17 @@ class AppVersionService extends Schmervice.Service {
             })
         }
 
+        // null-values are allowed for maxDhisVersion, so include these if filter is present
+        if (filters.getFilter('maxDhisVersion')) {
```

Comment:
We could push the `if` check to `applyVersionFilter`, allowing us to just do

```js
filters.applyVersionFilter(query, 'maxDhisVersion', {
    includeEmpty: true,
})
filters.applyVersionFilter(query, 'minDhisVersion')
```

Reply:
Interesting thought. I'm not sure that coupling the filter-names/logic to that function is worth it, as we could use it elsewhere.

Reply:
Nvm, I'm stupid. I understand what you mean - will add that instead!